day_2/prototypes.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+      <title>Object Oriented JavaScript</title>
+  </head>
+  <body>
+    <script type="text/Javascript">
+
+function getSum(num1, num2){
+  return num1 + num2
+}
+document.write("num of arguments " + getSum.length + "<br />");
+
+function Mammal(name){
+  this.name = name;
+  this.getInfo = function(){
+    return "The mammals name is " + this.name;
+  }
+}
+Mammal.prototype.sound = "Grrrr";
+
+Mammal.prototype.makeSound = function(){
+  return this.name + " says " + this.sound;
+}
+
+var grover = new Mammal("Grover");
+
+document.write(grover.makeSound() + "<br />")
+
+for(var prop in grover){
+  document.write(prop + " : " + grover[prop] + "<br />");
+}
+
+document.write("name Property of grover : " + grover.hasOwnProperty("name") + "<br />");
+
+document.write("sound Property of grover : " + grover.hasOwnProperty("sound") + "<br />");
+
+Array.prototype.inArray = function inArray(value){
+  for (var i = 0; i < this.length; i++){
+    if(this[i]=== value){
+      return true;
+    }
+  }
+  return false;
+}
+
+var sampArray = [1,2,3,4,5];
+
+document.write("3 in array " + sampArray.inArray(3) + "<br />")
+
+
+
+
+
+
+
```

Comment:
try to get into the habit of eliminating unnecessary white spaces before you make your commits/ PRs. it will help you in the future =D